include/sdf/CMakeLists.txt

```diff
@@ -1,73 +1,4 @@
-include (${sdf_cmake_dir}/SDFUtils.cmake)
+configure_file(config.hh.in
+  include/sdf/config.hh)
```

Comment:
this `configure_file` call can be removed since it is already handled by `ign_install_all_headers`

* https://github.com/ignitionrobotics/ign-cmake/blob/ign-cmake2/cmake/IgnUtils.cmake#L737-L755

Reply:
This is copied from https://github.com/ignitionrobotics/sdformat/blob/main/cmake/sdf_config.h.in which does not have include headers as well :thinking: . Maybe it's because it only contains `#define` so it doesn't matter if it is included multiple times?

Reply:
removed in 8b65844